events/chicago-workshop-2016/code-of-conduct-chicago-workshop-2016.md

```diff
@@ -0,0 +1,55 @@
+# Code of Conduct for INN Conferences and Meetings
```

Comment:
we don't need to include this here because it's already in the main events folder